tests/enhancer/gha/test_gainrule_camfit.py

```diff
@@ -0,0 +1,215 @@
+"""Tests for enhancer.gha.gainrule_camfit module"""
+import numpy as np  # noqa: I001
```

Comment:
Strange that `isort` is complaining about the import order here, its the same as `test_audiogram.py`. :thinking:

Reply:
Yep, not sure what was going on here. Will revisit before coming out of draft PR mode.